catalogs/catalog-jdbc-doris/src/main/java/com/datastrato/gravitino/catalog/doris/operation/DorisTableOperations.java

```diff
@@ -216,11 +220,60 @@ private static void appendPartitionSql(
           "The partition field must be one of the columns");
 
       String partitionColumn = BACK_QUOTE + rangePartition.fieldName()[0] + BACK_QUOTE;
-      // TODO we currently do not support pre-assign partition when creating range partitioning
-      partitionSqlBuilder.append(partitionColumn).append(") () ");
+      partitionSqlBuilder.append(partitionColumn).append(") ").append(NEW_LINE).append("(");
+
+      RangePartition[] assignments = rangePartition.assignments();
```

Comment:
Could you split the current `appendPartitionSql` method into several shorter methods? there are too many codes in one method

Reply:
This is already fixed.